modules/standard/FileSystem.chpl

```diff
@@ -27,7 +27,7 @@ use Error;
    Note: this is not safe within a parallel context.  A chdir call in one task
    will affect the current working directory of all tasks for that locale.
 */
-proc locale.chdir(out err: syserr, name: string) {
+proc locale.chdir(out error: syserr, name: string) {
```

Comment:
The use of the argument will probably need to be updated (though I'm sure testing will tell you that shortly)